### PR TITLE
Convert soql response to dataframe (READY FOR REVIEW)

### PR DESCRIPTION
**Pull request by rroig-at-wiris**
- Add function to convert soql response to Pandas DataFrame 
- Add test for soql_response_to_dataframe function

### CODE REVIEW

File: santoku/salesforce/lightning.py [hunk 1]
```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
...
         return records
 
+    def soql_response_to_dataframe(
+        self,
+        response: Dict[str, Any],
+        column_mapping: Dict[str, str] = None,
+        drop_columns_containing: str = None,
+    ) -> pd.DataFrame:
+        """
+        Flattens a SOQL response and converts it into a pandas DataFrame.
```
Comment (dmas-at-wiris) [inline santoku/salesforce/lightning.py]: Flattens **an** SOQL...

File: santoku/salesforce/lightning.py [hunk 2]
```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
...
+
+        To flatten the response json we use the function json_normalize from Pandas [1]
+
+        Parameters
```
Comment (dmas-at-wiris) [inline santoku/salesforce/lightning.py]: To write this doctoring take as an example the previous function docstring. Here we are missing, for example, some description in the parameters.

File: santoku/salesforce/lightning.py [hunk 3]
```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
...
+    def soql_response_to_dataframe(
+        self,
+        response: Dict[str, Any],
+        column_mapping: Dict[str, str] = None,
+        drop_columns_containing: str = None,
+    ) -> pd.DataFrame:
+        """
+        Flattens a SOQL response and converts it into a pandas DataFrame.
+
+        To flatten the response json we use the function json_normalize from Pandas [1]
```
Comment (dmas-at-wiris) [inline santoku/salesforce/lightning.py]: The response **JSON** object is flattened using the Pandas json_normalize function [1]

File: santoku/salesforce/lightning.py [hunk 4]
```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
...
+        pd.DataFrame
+            DataFrame with the flattened SOQL response.
+
+        References
+        ----------
+        [1] :
+            https://pandas.pydata.org/docs/reference/api/pandas.json_normalize.html
+
+        """
+        import pandas as pd
```
Comment (dmas-at-wiris) [inline santoku/salesforce/lightning.py]: remove this, not necessary any more

File: santoku/salesforce/lightning.py [hunk 5]
```diff
@@ -526,6 +535,55 @@ def do_query_with_SOQL(self, query: str) -> List[Dict[str, Any]]:
             records.extend(response_dict["records"])
         return records
 
+    def soql_response_to_dataframe(
```
Comment (dmas-at-wiris) [inline santoku/salesforce/lightning.py]: Shall this method be static?

**Review by dmas-at-wiris — CHANGES_REQUESTED** [review]
@HENRYQIULO why was everything reformated? Did we change anything from black?

**Comment by rroig-at-wiris** [comment]
> @HENRYQIULO why was everything reformated? Did we change anything from black?
@dmas-at-wiris The last version of black re-implemented support for explicit trailing commas. See reference [here](https://github.com/psf/black/blob/master/CHANGES.md#black-1). With the previous version (19.10b0), having the trailing comma, black didn't split the line. However, with the newer version (20.8b0), it does so. Hence, we will remove the trailing commas to avoid the split.

**Review by hqiu-at-wiris — APPROVED** [review]

**Pull request merged**